### ISSUE TITLE
Sync BatchNorm costs lots of time and cause code unworkable

### ISSUE THREAD

**Issue by ppbangKGT** [JMoonr/LATR#17]
Hi, thanks for your nice work!
When I run your code, there met a problem.
Because my device is unable to connect to the internet, I download the resnet50 pth pretrained on ImageNet instead of using the mm-pretrain model. This means I change the encoder config `init_cfg=dict(type='Pretrained', checkpoint='torchvision://resnet50')` to `init_cfg=dict(type='Pretrained', checkpoint='/data1/pbw/LATR-main/resnet50-19c8e357.pth')`
By doing so, I successfully run the code. But it stops at operation "Convert model with Sync BatchNorm", and cannot go on. I have taken a screenshot as follows.
![1705052282934](https://github.com/JMoonr/LATR/assets/76954965/1d43ed32-cfab-4301-9fd7-39709a8fdaaa)
Could you give me some advice? Thanks for your time.

**Comment by zjulabwjt** [comment]
> Hi, thanks for your nice work! When I run your code, there met a problem. Because my device is unable to connect to the internet, I download the resnet50 pth pretrained on ImageNet instead of using the mm-pretrain model. This means I change the encoder config `init_cfg=dict(type='Pretrained', checkpoint='torchvision://resnet50')` to `init_cfg=dict(type='Pretrained', checkpoint='/data1/pbw/LATR-main/resnet50-19c8e357.pth')` By doing so, I successfully run the code. But it stops at operation "Convert model with Sync BatchNorm", and cannot go on. I have taken a screenshot as follows. ![1705052282934](https://private-user-images.githubusercontent.com/76954965/296194502-1d43ed32-cfab-4301-9fd7-39709a8fdaaa.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjA0MTcyOTUsIm5iZiI6MTcyMDQxNjk5NSwicGF0aCI6Ii83Njk1NDk2NS8yOTYxOTQ1MDItMWQ0M2VkMzItY2ZhYi00MzAxLTlmZDctMzk3MDlhOGZkYWFhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MDglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzA4VDA1MzYzNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWUyZjkwZjc3OTVkNmQzZTgwNGJkMjA5MGNjZWQxZjQ4MTBiZTZjODg1NmM2ZjU4ZWE2MDg0OGM5YWQxYmRiMWImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.Zc6TZtveiaJu_6ILoS_HHLPxsQzmGdpNsLX390A31C0) Could you give me some advice? Thanks for your time.

have you solved this problem?
